### PR TITLE
flaky-test-reporter: add unit tests for GitHub issue flag

### PR DESCRIPTION
**Pull request by TrevorFarrelly**
Blocked on #813, will not compile until that is merged.

/hold
/lint

### CODE REVIEW

File: tools/flaky-test-reporter/github_issue_test.go [hunk 1]
```diff
@@ -79,9 +79,10 @@ func createNewIssue(fgi *GithubIssue, title, body, testStat string) (*github.Iss
 	return issue, comment
 }
 
-func createRepoData(passed, flaky, failed, notenoughdata int, startTime int64) *RepoData {
+func createRepoData(passed, flaky, failed, notenoughdata int, createissueflag bool, startTime int64) *RepoData {
```
Comment (srinivashegde86) [inline tools/flaky-test-reporter/github_issue_test.go]:
```suggestion
func createRepoData(passed, flaky, failed, notenoughdata int, createissue bool, startTime int64) *RepoData {
```

File: tools/flaky-test-reporter/github_issue_test.go [hunk 2]
```diff
@@ -105,15 +106,18 @@ func createRepoData(passed, flaky, failed, notenoughdata int, startTime int64) *
 func TestCreateIssue(t *testing.T) {
 	datas := []struct {
 		passed, flaky, failed, notenoughdata int
+		createissueflag											 bool
```
Comment (srinivashegde86) [inline tools/flaky-test-reporter/github_issue_test.go]:
```suggestion
		createissueflag bool
```

Comment (TrevorFarrelly) [reply tools/flaky-test-reporter/github_issue_test.go]: ran gofmt, should be standard formatting now.

File: tools/flaky-test-reporter/github_issue_test.go (hunk 1 above)
Comment (TrevorFarrelly) [reply tools/flaky-test-reporter/github_issue_test.go]: fixed.

**Comment by srinivashegde86** [comment]
waiting on Chao to take a look and lgtm

**Pull request merged**